zanata-war/src/main/webapp-jboss/WEB-INF/web.xml

```diff
@@ -148,6 +148,10 @@
   </listener>
 
   <listener>
+    <listener-class>org.zanata.servlet.ZanataDatabaseMetadataServletListener</listener-class>
```

Comment:
It would be more trendy to use `javax.servlet.annotation.WebListener` on the class itself.